waspc/examples/todo-typescript/main.wasp

```diff
@@ -41,7 +41,7 @@ app TodoTypescript {
     system: PostgreSQL
   },
   webSocket: {
-    fn: import { webSocketFn } from "@src/websocket/index.js",
```

Comment:
Removing extensions and it keeps working 🎉

Reply:
We have an issue with a different behaviour in our LSP and what our code will tolerate.

| <img width="571" alt="Screenshot 2024-02-05 at 17 06 53" src="https://github.com/wasp-lang/wasp/assets/2223680/cf8afe11-da75-4992-900f-367c42c9e5b2"> |
| --- |
| LSP doesn't recognise imports from folders - this will run time though |

| <img width="593" alt="Screenshot 2024-02-05 at 17 08 04" src="https://github.com/wasp-lang/wasp/assets/2223680/b991c557-5c9b-4a49-944b-a7298b1d1a4e"> |
| --- |
| Adding `index` helps - this will also run fine |

Reply:
Yeah this is tricky. DO you mind opening an issue for it? You can mark it with `restructuring`. I think we can handle this one after 0.12 is out.

Reply:
I've created an issue for us to tackle this later https://github.com/wasp-lang/wasp/issues/1717